publish.sh

```diff
@@ -37,6 +37,9 @@ VERSION=$(cat ./VERSION)
 [[ "${PREFIX}" == */ ]] && PREFIX="${PREFIX%?}"
 PREFIX_AND_VERSION=${PREFIX}/${VERSION}
 
+# Remove invalid character at the end of PREFIX_AND_VERSION
+PREFIX_AND_VERSION=${PREFIX_AND_VERSION::-1} 
```

Comment:
This changes will always remove one character from the variable value, which will break when the last character is a valid component of the version number. This problem is not manifest in our Linux (Amazon Linux 2) build environments, and  I suspect the problem is specific to the WSL build environment.. perhaps on Windows the VERSION file has an extra control character that is not present in Linux.   
Can you implement a more robust solution that removed only control characters or whitespace, but will not remove valid characters, and verify that your changes do not have adverse effects on Linux build environments..  See [here](https://www.cyberciti.biz/faq/bash-remove-whitespace-from-string/) for suggestions.
Thanks!